x/assets/keeper/operator_asset.go

```diff
@@ -96,7 +96,7 @@ func (k Keeper) UpdateOperatorAssetState(ctx sdk.Context, operatorAddr sdk.Addre
 // IteratorAssetsForOperator iterates all assets for the specified operator
 // if `assetsFilter` is nil, the `opFunc` will handle all assets, it equals to an iterator without filter
 // if `assetsFilter` isn't nil, the `opFunc` will only handle the assets that is in the filter map.
-func (k Keeper) IteratorAssetsForOperator(ctx sdk.Context, operator string, assetsFilter map[string]interface{}, opFunc func(assetID string, state *assetstype.OperatorAssetInfo) error) error {
+func (k Keeper) IteratorAssetsForOperator(ctx sdk.Context, isUpdate bool, operator string, assetsFilter map[string]interface{}, opFunc func(assetID string, state *assetstype.OperatorAssetInfo) error) error {
```

Comment:
`IteratorAssetsForOperator` => `IterateAssetsForOperator`? since this function iterates over assets for operator

Reply:
yeah

Reply:
it will be resolved in the PR regarding registering tokens and client chains.